### ISSUE TITLE
Chart title links pointing to Twitter search

### ISSUE THREAD

**Issue by fabiogiglietto** [cantino/huginn#1552]
I'm using the PeakDetectorAgent to monitor a stream of Facebook post activities. I'm still testing, it but the agent seems to be doing its job. However, I've noticed a small bug in the agent summary page. The link in the charts title always point to a twitter search. It would be great to let the user customize that url.

Best, Fabio

**Comment by dsander** [comment]
You are right, that would be a nice addition, I assume you would like the base URL to be configurable in the agent with the term being dynamically replaced like it is done at the moment for the twitter search term?

**Comment by fabiogiglietto** [comment]
Yes :)

Il dom 19 giu 2016, 12:08 AM Dominik Sander <notifications@github.com> ha
scritto:
